pimd/pim_cmd.c

```diff
@@ -508,7 +508,7 @@ static void igmp_show_interfaces(struct pim_instance *pim, struct vty *vty,
 		if (!pim_ifp)
 			continue;
 
-		for (ALL_LIST_ELEMENTS_RO(pim_ifp->socket_list, sock_node,
+		for (ALL_LIST_ELEMENTS_RO(pim_ifp->gm_socket_list, sock_node,
```

Comment:
In what context have you changed this? Are you following the convention of our PRs which we have raised?

Reply:
I do not understand your question here. This is feedback on one of your PRs. It matches the context and convention of it.